app/web/typo3conf/ext/theme/Resources/Private/Partials/Page/Development.html

```diff
@@ -0,0 +1,7 @@
+{namespace s7=Sup7even\Theme\ViewHelpers}
+
+<f:if condition="{s7:context()} != 'Production'">
```

Comment:
Maybe it's better to set the state on/off for this feature via fluid setting in TypoScript?